otelcollector/deploy/addon-chart/azure-monitor-metrics-addon/templates/ama-metrics-deployment.yaml

```diff
@@ -62,7 +62,11 @@ spec:
             - name: AZMON_COLLECT_ENV
               value: "false"
             - name: AZMON_OPERATOR_ENABLED
+            {{- if or (eq .Values.AzureMonitorMetrics.ArcExtension false) (eq .Values.AzureMonitorMetrics.ArcEnableOperator true) }}
```

Comment:
Is the ArcExtension variable always set to either true or false? Can this be nil?

Reply:
This is a good point, thanks Rashmi. Updated this to work in case it is nil